### PR TITLE
[Dicom Archive - View Details] module subpage help text

### PR DESCRIPTION
**Pull request by knoll-alex**
resurrected and updated help text for the View Details subpage of the DICOM Archive

#### Testing instructions

1. review copy
2. check screenshot for rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

Screenshot 2019-11-25 20 12 56

### CODE REVIEW

**Comment by johnsaigle** [comment]
Failed due to #5719. Restarting Travis.

File: modules/dicom_archive/help/viewDetails.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# DICOM Archive: Tarchive Metadata
+# View Details
 
-The “View Details” page of the DICOM Archive displays metadata for a given DICOM tarchive (compressed package of DICOM files). 
+You've arrived at this page by clicking on **View Details** from the DICOM Archive. This page displays metadata for your selected DICOM tarchive (compressed package of DICOM files).
```
Comment (driusan) [inline modules/dicom_archive/help/viewDetails.md]: > You've arrived at this page by clicking on **View Details** from the DICOM Archive.

Seems pretty speculative.

Comment (knoll-alex) [reply modules/dicom_archive/help/viewDetails.md]: true. removed

File: modules/dicom_archive/help/viewDetails.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
-# DICOM Archive: Tarchive Metadata
+# View Details
 
-The “View Details” page of the DICOM Archive displays metadata for a given DICOM tarchive (compressed package of DICOM files). 
+This page displays metadata for a selected DICOM tarchive (compressed package of DICOM files). 
 
-All DICOM header fields and other metadata from the DICOM dataset are presented in the page, including information about the candidate, number of acquisitions, where the files were acquired, and all files contained within the given archive.
+Here, you'll fine all DICOM header fields and other metadata from the DICOM dataset, including information about the candidate, number of acquisitions, where the files were acquired, and all files contained within the given archive.
```
Comment (driusan) [inline modules/dicom_archive/help/viewDetails.md]:
```suggestion
Here, you'll find all DICOM header fields and other metadata from the DICOM dataset, including information about the candidate, number of acquisitions, where the files were acquired, and all files contained within the given archive.
```

typo

Comment (knoll-alex) [reply modules/dicom_archive/help/viewDetails.md]: thx

**Pull request merged**